requests/models.py

```diff
@@ -918,7 +918,7 @@ def links(self):
     def raise_for_status(self):
         """Raises :class:`HTTPError`, if one occurred."""
 
-        http_error_msg = ''
+        http_error_msg, reason = '', ''
```

Comment:
This change appears to be unrelated to the range change.
Was it suggested by a linting tool?